### ISSUE TITLE
Blacklist breaks module with latest nvidia drivers

### ISSUE THREAD

**Issue by willmcenaney** [tseliot/nvidia-prime#20]
Looks like nvidia may have changed their module config.
`alias nvidia off` no longer works in the blacklist

Similar to the below, but only started affecting us recently.
https://askubuntu.com/questions/1084829/modprobe-nvidia-throws-error-could-not-insert-off

**Comment by tseliot** [comment]
@willmcenaney do you mean that nvidia is loaded regardless?

**Comment by willmcenaney** [comment]
No, the 'could not insert 'off'' error happens and stops `nvidia-smi` from running correctly (unknown other effects)

**Comment by tseliot** [comment]
why is the driver blacklisted at all?

**Comment by willmcenaney** [comment]
`prime-select intel` adds the blacklist (hence me raising the issue here) - apologies I was a little sparse on detail i've pieced it together from a colleague

**Comment by tseliot** [comment]
@willmcenaney my question is why are you trying to load the nvidia module if you selected the "intel" profile?

**Comment by willmcenaney** [comment]
We're not - but because the blacklist errors it results in a broken `nvidia-smi`.  Running `modprobe nvidia` just allows replication of the error (as `nvidia-smi` just tells you it can't communicate with the drivers).

Our end goal is to install nvidia drivers for parallel processing only.  We are running `prime-select intel` to do so, however `nvidia-smi` outputs `NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.`

**Comment by tseliot** [comment]
Then "on-demand" mode is what you want, rather than "intel". The former doesn't blacklist the driver or use it to render the desktop.
